lib/server/game.js

```diff
@@ -13,6 +13,10 @@ class Game {
     this.spectators = [];
     this.gameMode = gameMode;
     this.engine = new GameEngine(gameMode.engineOptions);
+
+    const dummy = () => null;
```

Comment:
Lo-Dash has [noop](https://lodash.com/docs/4.17.4#noop) utility function for this purpose.